src/OptPass.cc

```diff
@@ -60,9 +71,134 @@ bool OptPass::runOnModule(Module&)
 
   outs() << "Got policy file!\n";
 
-  return false;
-}
+	std::map<CallInst*, std::vector<Policy::Direction>> callInstsMap;
+		ErrorOr<std::unique_ptr<PolicyFile>> p = PolicyFile::Open();
+		if (p.getError()) {
+			errs() << p.getError().message() << "\n";
+			errs() << "error when retrieving Policy File \n";
+			return false;
+		}
+		assert(*p);
+		Policy& policy = **p;
 
+    for (Module::iterator i = module.begin(), e = module.end(); i!=e; i++) {
```

Comment:
Could this bit (finding all of the calls) be refactored out into a separate method?